confdgnmi/testtool/gNMIRobotLibrary.py

```diff
@@ -10,8 +11,15 @@ class gNMIRobotLibrary(ABC):
     last_exception: Optional[Exception] = None
 
     """ Common gNMI related functionality used across Robot tests and all libraries inheriting. """
-    def __init__(self) -> None:
+    def __init__(self, disable_extra_logs = True) -> None:
         self._client: Optional[ConfDgNMIClient] = None
+        if disable_extra_logs:
```

Comment:
Yes, but .... 

I think right log levels in `confd_gnmi_client.py` should be enough. 
Also, does not robot support not to dump logs at all (only when troubleshooting)? 
Informative text about progress, processing should be part of test itself.